test/e2e/e2e_test.go

```diff
@@ -135,3 +137,43 @@ func TestMain(m *testing.M) {
 func TestE2E(t *testing.T) {
 	RunE2ETests(t)
 }
+
+var _ = ginkgo.ReportAfterSuite("Kubernetes e2e suite report", func(report ginkgo.Report) {
+	var f *os.File
+	var err error
+	// The DetailsRepoerter will output details about every test (name, files, lines, etc) which helps
+	// when documenting our tests.
+	if len(framework.TestContext.SpecSummaryOutput) > 1 {
```

Comment:
current code says it runs if > 0 
```
	// The DetailsRepoerter will output details about every test (name, files, lines, etc) which helps
	// when documenting our tests.
	if len(framework.TestContext.SpecSummaryOutput) > 0 {
		r = append(r, e2ereporters.NewDetailsReporterFile(framework.TestContext.SpecSummaryOutput))
	}
```

anyway, we can remove an indentation level if we return fast instead
```
if len(framework.TestContext.SpecSummaryOutput) <= 0 {
   return
}
// reporter here
```

Reply:
Done